entrypoint.sh

```diff
@@ -70,6 +70,14 @@ function setup_environment() {
       export ENABLE_SCORECARD="true"
   fi
 
+  export GITHUB_AUTH_TOKEN=$INPUT_REPO_PAT
```

Comment:
By using different names it's possible that logs will be confusing; can we align these?

```suggestion
  export GITHUB_AUTH_TOKEN=$INPUT_GITHUB_AUTH_TOKEN
```